### PR TITLE
Build up a timeline of events for incident reports

### PR DESCRIPTION
**Pull request by milesbxf**
This adds a new model, `TimelineEvent`, and corresponding API to manage it.

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -5,7 +5,8 @@ python_paths = demo
 ; point at demo app Django settings
 DJANGO_SETTINGS_MODULE = demo.settings.dev
 
-; addopts = -n 8
+; Set up DB by inspecting models rather than running migrations
+addopts = --nomigrations
```
Comment (mattrco) [inline pytest.ini]: Is this so we can run tests on a new model without creating the migration first?

Comment (milesbxf) [reply pytest.ini]: Whoops - good point, I should create the migrations

File: response/core/serializers.py [hunk 2]
```diff
@@ -10,6 +10,12 @@ class Meta:
         fields = ("app_id", "external_id", "display_name")
 
 
+class TimelineEventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = TimelineEvent
+        fields = ("timestamp", "text", "event_type", "metadata")
```
Comment (mattrco) [inline response/core/serializers.py]: Not a blocker, but will metadata be mapped back to a string or will it be a tuple index?

Comment (mattrco) [reply response/core/serializers.py]: Ah, I see it's stored as a text field 👍

Comment (milesbxf) [reply response/core/serializers.py]: It _should_ be serialized as nested JSON in the API response.

In the model it corresponds to a JSONField, which is only supported in Postgres - so this is tricky to automatically test

**Pull request merged**